lib/hash.py

```diff
@@ -143,18 +143,18 @@ def encode(be_bytes):
         return txt[::-1]
 
     @staticmethod
-    def decode_check(txt):
+    def decode_check(txt, hash_fn=double_sha256):
```

Comment:
For this and encode_check please insert a *:

def decode_check(txt, *, hash_fn=double_sha256)